src/display/editor/drawers/freedraw.js

```diff
@@ -591,17 +246,60 @@ class FreeOutliner {
         );
       }
     }
-    buffer.push(`L${bottom[4]} ${bottom[5]} Z`);
+
+    this._toSVGPathStart(buffer);
 
     return buffer.join(" ");
   }
 
+  _toSVGPathTwoPoints() {
```

Comment:
Can we make this method actually private instead?
```suggestion
  #toSVGPathTwoPoints() {
```